platform/viewer/src/connectedComponents/ViewerRetrieveStudyData.js

```diff
@@ -83,18 +85,56 @@ class ViewerRetrieveStudyData extends Component {
     });
   }
 
+  _handleSeriesLoadResult(error, studyMetadata, series) {
```

Comment:
This signature is weird. Shouldn't it be `studyMetadata`, `series`, `error` as error is optional?

Reply:
I tried to follow the old JS standard for callbacks hehehe...
If there is an error, none of the other parameters should be used.